scripts/test_hive.sh

```diff
@@ -30,14 +30,15 @@ set -e
 hdfs dfs -rm -r -f hdfs://localhost:8020/sqlflow
 hdfs dfs -mkdir -p hdfs://localhost:8020/sqlflow
 export SQLFLOW_HIVE_LOCATION_ROOT_PATH=/sqlflow
+# enable go mod
+export GO111MODULE=on
```

Comment:
@typhoonzero @tonyyang-svail Can we remove the settings of `GO111MODULE=on`  if we install Go 1.13 into our Docker image by editing the following code snippet?

https://github.com/sql-machine-learning/sqlflow/blob/2dc6657bc4095852c3cc666aadfbf0b6efca175f/scripts/build_docker_image.sh#L47-L51

I ask because it seems from https://blog.golang.org/using-go-modules that since 1.13, the module mode is by default on.